filament/src/PostProcessManager.cpp

```diff
@@ -1272,7 +1272,7 @@ FrameGraphId<FrameGraphTexture> PostProcessManager::dof(FrameGraph& fg,
 
 FrameGraphId<FrameGraphTexture> PostProcessManager::bloomPass(FrameGraph& fg,
         FrameGraphId<FrameGraphTexture> input, TextureFormat outFormat,
-        View::BloomOptions& bloomOptions, float2 scale) noexcept {
+        View::BloomOptions& bloomOptions, float2 scale, bool disableFeedbackLoops) noexcept {
```

Comment:
I think this state should exist in the postprocess manager.  You can initilize it in `PostProcessManager::init`. This way we don't have to pass the flag in all APIs.

Reply:
Ah, good idea, done.